py4DSTEM/process/diffraction/crystal.py

```diff
@@ -345,9 +361,7 @@ def from_unitcell_parameters(
 
         return Crystal.from_pymatgen_structure(structure)
 
-    def setup_diffraction(
-        self, accelerating_voltage: float
-    ):
+    def setup_diffraction(self, accelerating_voltage: float):
```

Comment:
I still think we should be able to skip specifying voltage here in many quick workflows - perhaps instead specify a default option for crystal on instantiation?

Reply:
OK I made it default to 300 kV if none is set, and give a warning in `generate_diffraction_pattern`